pkg/recipe/recipe_test.go

```diff
@@ -58,3 +59,15 @@ func TestRecipeRenderChecksums(t *testing.T) {
 		t.Errorf("Expected checksum %s for content %s to match %s", readme.Content, readme.Checksum, sumWithAlgo)
 	}
 }
+
+func TestRecipeDefaults(t *testing.T) {
+	recipe := Recipe{}
+	recipeStr := `name: foo
```

Comment:
Should this function test the recipe which is created by `jalapeno create`? Or what do you mean by recipe "defaults" in this context?

Reply:
I think this had a test for what the value of the ignore flag for README.md was after unmarshaling. Right now it doesn't test anything so I'll remove it. Good catch!